snooty.toml

```
@@ -25,10 +25,11 @@ atlas = "MongoDB Atlas"
 targets = ["motor.txt"]
 variant = "warning"
 value = """
-    As of May 14th, 2025, Motor has been sunset in favor of the GA release of the PyMongo Async API in Pymongo. \
-    No new features will be added to Motor. Motor will be deprecated on May 14th, 2026, \
-    and no further bug fixes will be released after deprecation, with the exception of critical bug fixes, which will be released until May 14th, 2027. \
-    We strongly recommend that Motor users migrate to PyMongo Async while Motor is still supported. \
-    To learn how to migrate from Motor to the PyMongo Async API, see the \
-    `Migrate to PyMongo Async guide <https://www.mongodb.com/docs/languages/python/pymongo-driver/current/reference/migration/>`__.
+    As of May 14, 2025, Motor is deprecated in favor of the GA release of the PyMongo Async
+    API in the PyMongo library. No new features will be added to Motor, and we will only
+    provide bug fixes until it reaches end of life on May 14, 2026. After that, we will only
+    fix critical bugs until final support ends on May 14, 2027. We
+    strongly recommend migrating to the PyMongo Async API while Motor is still supported.
+    For more information on migrating, see the `Migrate to PyMongo Async <https://www.mongodb.com/docs/languages/python/pymongo-driver/current/reference/migration/>`__
+    guide in the PyMongo documentation.
```

I: [about/on](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-about:~:text=Don%27t%20use%20on%20to%20mean%20about.)

```suggestion
    For more information about migrating, see the `Migrate to PyMongo Async <https://www.mongodb.com/docs/languages/python/pymongo-driver/current/reference/migration/>`__
    guide in the PyMongo documentation.
```

can this be its own paragraph?